godtools/App/Features/Onboarding/OnboardingQuickStartView.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  OnboardingQuickStartView.swift
+//  godtools
+//
+//  Created by Robert Eldredge on 11/8/21.
+//  Copyright © 2021 Cru. All rights reserved.
+//
+
+import UIKit
+
+class OnboardingQuickStartView: UIViewController {
+    
+    enum DismissOnboardingTutorialType {
```

Comment:
This type doesn't look right to me here because DismissOnboardingTutorialType is related to steps to take in the Flow.  If this type were used by this View that would make more sense to have it declared here.  See my comment in FlowStep ```dismissOnboardingTutorial```.